src/main/java/core/basesyntax/service/DataConverterImpl.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> lines) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        for (int i = 1; i < lines.size(); i++) {
+            String line = lines.get(i);
+            String[] split = line.split(",");
+
+            String operationValue = split[0].trim();
+            String fruit = split[1].trim();
+            int amount = Integer.parseInt(split[2].trim());
+
```

Comment:
```suggestion
            String operationValue = split[0].trim();
            String fruit = split[1].trim();
            int amount = Integer.parseInt(split[2].trim());
```
Don’t  use trim here